### PR TITLE
allow search less than three characters

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**
Signed-off-by: walid-sahnoun <walid.sahnoun@rte-france.com>

### CODE REVIEW

**Review by laurgarn — APPROVED** [review]

File: src/components/ElementSearchDialog/element-search-dialog.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import PropTypes from 'prop-types';
 import SearchIcon from '@mui/icons-material/Search';
 import { useIntl } from 'react-intl';
 
-const TERM_MIN_SIZE_BEFORE_SEARCH = 3;
+const TERM_MIN_SIZE_BEFORE_SEARCH = 1;
```
Comment (SlimaneAmar) [inline src/components/ElementSearchDialog/element-search-dialog.js:14]: Why do you keep this constant ?
I think we can delete it

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/components/ElementSearchDialog/element-search-dialog.js (hunk 1 above)
Comment (Walid-Sahnoun) [reply src/components/ElementSearchDialog/element-search-dialog.js:14]: well i tried to removed but we can't because when we search with one character and removed there is request to the server with empty term

Comment (SlimaneAmar) [reply src/components/ElementSearchDialog/element-search-dialog.js:14]: you need only test if the term is empty or not

**Review by SlimaneAmar — APPROVED** [review]

File: src/components/ElementSearchDialog/element-search-dialog.js (hunk 1 above)
Comment (Walid-Sahnoun) [reply src/components/ElementSearchDialog/element-search-dialog.js]: done
